engine/src/main/java/org/terasology/rendering/dag/nodes/AlphaRejectBlocksNode.java

```diff
@@ -190,4 +206,35 @@ public void process() {
 
         PerformanceMonitor.endActivity();
     }
+
+    @Override
+    public void propertyChange(PropertyChangeEvent event) {
+        if (event.getOldValue() != event.getNewValue()) {
```

Comment:
Yes, you are absolutely correct.

I've added a comment to all these functions implying that it only gets called when the property is changed, since we want to make things as explicit as possible. I hope that's not too redundant for your tastes?